src/data-table/utils/formatting.tsx

```diff
@@ -40,3 +36,12 @@ export function formatRowValue<T extends ReferenceObject>(
 }
 
 export const defaultRowConfig = () => ({} as RowConfig);
+
+function isStringDate(value: any): boolean {
+    // Avoid dubious positives by skipping strings that do not contain at least year, month and day.
+    if (typeof value === "string" && value.length >= "YYYY-MM-DD".length) {
+        return moment(value, moment.ISO_8601, true).isValid();
```

Comment:
Not completely sure about this check.

Would it make sense to check for multiple valid formatting in strict mode instead of ISO_8601:

``formats = [DD/MM/YYYY, DD-MM-YYYY, MM/DD/YYYY, MM-DD-YYYY]``

And use ``moment(value, format, true).isValid()``?

Reply:
As discussed earlier this week, adding the check from ``creationData()``